keyboards/tn_alice/rules.mk

```diff
@@ -0,0 +1,10 @@
+# Wildcard to allow APM32 MCU

```

Comment:
Are you actually using APM32 or is this copy-paste?

Reply:
I'm actually use APM32. Should I delete this line?

Reply:
Hi @zvecr, I've deleted this line but the CI always says 'rules.mk requires formating'. Which sections should I format my rules.mk? Can you give me a sample rules.mk format?

Reply:
You still have Windows line endings in `rules.mk` (CRLF); you need to change them to Unix line endings (LF).

Reply:
If you are actually using APM32, then you should not delete the line.

Reply:
Actually `DFU_SUFFIX_ARGS` is needed only for chips like APM32F072, which have a builtin USB DFU bootloader in ROM — in that case the USB VID:PID pair is provided by the vendor-specific bootloader, and those IDs are different between STM32 and APM32 implementations.  However, in case of STM32F103 and APM32F103 the chip does not have a builtin USB DFU bootloader in ROM, and the user bootloader can be the same on both of those chips, so the VID:PID pair of the bootloader does not change when you are using the APM32F103 clone instead of the original STM32F103 chip, and specifying `DFU_SUFFIX_ARGS` is not required.

Even more, this particular board uses the `uf2boot` bootloader, therefore `DFU_SUFFIX_ARGS` is not used at all.

Reply:
Ah good point on the bootloader, ignore my suggestion.